auth/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: terraform Authentication
+description: Authenticates with terraform
+inputs:
+  checkout-repo:
```

Comment:
I don't think we need checkout repo here at all since this doesn't work with repo objects.

Reply:
Wouldn't you expect the `.terraformrc` file to be under git control if it already exists?

Reply:
Ah, I misunderstood. I have removed the checkout-repo input and optional Checkout step.